app/controllers/orders_controller.rb

```diff
@@ -18,6 +18,10 @@ def update
     end
   end
 
+  def show
+    @order = Order.find(session[:order_id])
```

Comment:
Why use session and not params to find the order?

Reply:
Fixed